src/test/suite/extensionFunctions.test.ts

```diff
@@ -34,6 +34,17 @@ suite('Extension Functions Test Suite', () => {
 		await cleanFiles();
 	});
 
+	test('open a named output channel and send text', () => {
+		const channelName = 'testOutputChannel';
+		let channel: vscode.OutputChannel | undefined = getCachedOutputChannel(channelName);
+		assert.strictEqual(channel, undefined);
+
+		extensionFunctions.openNamedOutputChannel(channelName, 'someText');
+		channel = getCachedOutputChannel(channelName);
+		assert.notStrictEqual(channel, undefined);
+		assert.strictEqual(channel?.name, channelName);
+	});
```

Comment:
the test is checking only the channelName, even if a new channel is created the test will pass.

Reply:
This test creates a channel underneath, so yes, it will pass if it finds the created channel. I don't see your point.

Reply:
oh wait, the test is not intended to check the cached method. I though t that as the assertions were all on the cached output channel.

would be nice to add some common use case in test:
- when command is called 2 times, checked that a single outputchannel is used
- check that the provided text is written in the outputchannel

Reply:
There is no way to retrieve the text written to an output channel. There is a feature request for that open though: https://github.com/microsoft/vscode/issues/65108

Reply:
given that you are creatign the output channel and in test, I think you can use spies

Reply:
tests have been added